aperturedb/Connector.py

```diff
@@ -182,6 +182,11 @@ def __del__(self):
             self.connected = False
 
     def _send_msg(self, data):
+        # aperturedb's param ADB_MAX_CONNECTION_MESSAGE_SIZE_MB = 256 by default
+        if len(data) > (256 * 2**20):
+            logger.warning(
+                "Message sent is larger than default for ApertureDB Server. Server may disconnect.")
```

Comment:
Curious to know why the word is "may"? I thought that it was a hard limit.
Should we just fail the query, and write a scenario in tests when this is triggered?

Reply:
it "may" disconnect because the server may use a different configuration than the default. If the server uses a different configuration than the default, the server will disconnect if the message size is larger than the set value (which can be more or less than 256MB).

Until we do a new client-server protocol, there is no way to communicate the limit to the client without breaking compatibility, that's why we are holding off the change.